# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nexuiz/cd_linux.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./cd_linux.c.orig	2005-06-04 13:48:36.000000000 +0200
+++ ./cd_linux.c	2005-06-04 13:50:24.000000000 +0200
@@ -20,7 +20,6 @@
 // Quake is a trademark of Id Software, Inc., (c) 1996 Id Software, Inc. All
 // rights reserved.
 
-#include <linux/cdrom.h>
 #include <sys/ioctl.h>
 
 #include <errno.h>
@@ -28,6 +27,8 @@
 #include <time.h>
 #include <unistd.h>
 
+#include <linux/cdrom.h>
+
 #include "quakedef.h"
 #include "cdaudio.h"
 
